src/docs/concepts-development/translations/index.md

```diff
@@ -0,0 +1,192 @@
+`sdk.assetByAlias`. TODO: CLARIFY!!
+
+## Translation format for editing translations in Console.
+
+A translation using the
+[React Intl formatMessage formatting](https://formatjs.io/docs/intl#formatmessage) 
+can, at its simplest, consist of a phrase.
+
+```js
+const translations = {
+  en: {
+    "ListingPage.bookingSubTitle": "Start by choosing your dates."
+  }
+}
+...
+
+const bookingSubTitle = intl.formatMessage({ id: 'ListingPage.bookingSubTitle' });
+```
+
+### Simple argument
+
+In addition, the format supports passing parameters as arguments to the
+translation string. Passing a
+[simple argument](https://formatjs.io/docs/core-concepts/icu-syntax/#simple-argument)
+allows showing context-specific information as a part of the translation
+string.
+
+```js
+// The parameter {title} will be replaced with whatever the listing's title is
+const translations = {
+  en: {
+    "ListingPage.bookingTitle": "Book {title}",
+  }
+}
+...
+// When we pass the listing's title in 'values' to the message, 
+// for a listing titled "Small cottage" the message will read "Book Small cottage"
+const bookingTitle = intl.formatMessage({
+  id="ListingPage.bookingTitle"
+  values={{ title: listing.title }}
+});
+
+```
+Do note that even if the translation message uses a simple argument, you can choose to not use it. For instance, you could replace the translation in the previous example with `"ListingPage.bookingTitle": "Book this listing"`. However, if you later decide you do want to use the title, it is recommended to double check the original translation file in your client application to see the names of the attributes available in the message.
+
+
+### plural
+
+One important factor in creating natural translations is handling
+pluralization in a text. The ICU format makes it possible to define
+different wordings for singular and plural options.
+
+```js
+/* When you use plural in the translation string, you will need to specify
+- the variable determining which option to use (here: 'count')
+- the pattern we are following (here: 'plural')
+- the options matching each alternative you want to specify (here: 'one' – there could be several options specified)
+- an 'other' option that gets used when none of the specified alternatives matches
+*/
+const translations = {
+  en: {
+    "ManageListingsPage.youHaveListings": "You have {count} {count, plural, one {listing} other {listings}}",
+  }
+}
+...
+  // When we pass 'count' as a value, the formatting determines which translation option, 'one' or the default,
+  // is used in the final message string.
+  const heading = (
+    <FormattedMessage
+      id="ManageListingsPage.youHaveListings"
+      values={{ count: pagination.totalItems }}
+    />
+
+  // If pagination.totalItems === 1, the message reads "You have 1 listing" / "Tienes 1 anuncio".
+  // For any other number, including zero, the message reads "You have X listings" / "Tienes X anuncios".
+)
+```
```

Comment:
When you use **plural** in the translation string, you will need to specify
- the variable determining which option to use (here: 'count')
- the pattern we are following (here: 'plural')
- the options matching each alternative you want to specify (here: 'one' – there could be several options specified)
- an 'other' option that gets used when none of the specified alternatives matches

Translation asset:
```json
{
  "ManageListingsPage.youHaveListings": "You have {count} {count, plural, one {listing} other {listings}}",
}
```

How it's used in your client app:
```js
  // When we pass 'count' as a value, the formatting determines which translation option, 'one' or the default,
  // is used in the final message string.
  const heading = (
    <FormattedMessage
      id="ManageListingsPage.youHaveListings"
      values={{ count: pagination.totalItems }}
    />
  // If pagination.totalItems === 1, the message reads "You have 1 listing" / "Tienes 1 anuncio".
  // For any other number, including zero, the message reads "You have X listings" / "Tienes X anuncios".
)
```